libs/coin-modules/coin-mina/CHANGELOG.md

```diff
@@ -0,0 +1,118 @@
+# @ledgerhq/coin-near
+
+## 0.3.12
```

Comment:
You need to be removed near-specific changelog below, it's done automatically

Reply:
Fixed!